README.md

```diff
@@ -1,22 +1,30 @@
 # ARM mbed OS
  
-mbed OS is an open-source embedded operating system designed specifically for the "things" in the Internet of Things (IoT). It includes all the features you need to develop a connected product based on an ARM Cortex-M microcontroller.
+mbed OS is an open-source embedded operating system designed for the "things" in the Internet of Things (IoT). mbed OS includes the features you need to develop a connected product using an ARM Cortex-M microcontroller.
  
-mbed OS accelerates the process of creating a connected product by providing a platform operating system that includes robust security foundations, standards based communication capabilities, built-in cloud management services, and drivers for sensors, I/O devices and connectivity. mbed OS is built as a modular, configurable software stack so that you can readily customize it to the device you're developing for, and reduce memory requirements by excluding unnecessary software components.
+mbed OS provides a platform that includes:
+ - Security foundations.
+ - Cloud management services.
+ - Drivers for sensors, I/O devices and connectivity.
+
+mbed OS is modular, configurable software that you can customize it to your device and to reduce memory requirements by excluding unused software.
+
  
-## Current release
+## Release Notes
+
+The [Release Notes](https://docs.mbed.com/docs/mbed-os-release-notes/en/latest/) detail the current release and previous versions.
+
+## Continuous Integration Status
 
-The current release, along with a selection of previous versions are detailed here:
-[Release Notes](https://docs.mbed.com/docs/mbed-os-release-notes/en/latest/)
+We run continuous integration on all of our branches and pull requests to verify the stability of mbed OS. The following are the Travis CI indicators for mbed OS.
+
+ - Master branch [![Master Branch CI Badge](https://travis-ci.org/ARMmbed/mbed-os.svg?branch=master)](https://travis-ci.org/ARMmbed/mbed-os)
+ - Latest release [![5.4 Branch CI Badge](https://travis-ci.org/ARMmbed/mbed-os.svg?branch=latest)](https://travis-ci.org/ARMmbed/mbed-os/branches)
```

Comment:
Can we remove the 5.4 hard code and say latest release so this doesnt need an update every few months? I'm sure it'll get forgotten.